src/Arcus.EventGrid.Publishing/CloudEvents/CloudEventBatchContent.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+// ReSharper disable once CheckNamespace
+namespace CloudNative.CloudEvents
+{
+    /// <summary>
+    /// Represents a <see cref="HttpContent"/> model containing a batch of <see cref="CloudEvent"/>s instances.
+    /// </summary>
+    public class CloudEventBatchContent : HttpContent
```

Comment:
Why are we re-inventing the wheel instead of building on top of the SDK?https://github.com/cloudevents/sdk-csharp#http---systemnethttphttpclient

Reply:
Not re-inventing, but building on top. The SDK doesn't support batched cloud events, so made this to support the current `HttpContent` implementation for cloud events.

Reply:
Makes sense, thanks for letting us know!

Reply:
Added issue for this: https://github.com/cloudevents/sdk-csharp/issues/44